src/mame/konami/konamim2.cpp

```diff
@@ -1423,20 +1427,21 @@ ROM_START( totlvicj )
 	ROM_REGION64_BE( 0x200000, "boot", 0 )
 	ROM_LOAD16_WORD( "623b01.8q", 0x000000, 0x200000, CRC(bd879f93) SHA1(e2d63bfbd2b15260a2664082652442eadea3eab6) )
 
-	ROM_REGION( 0x100000, "ymz", 0 ) /* YMZ280B sound rom on sub board */
+	ROM_REGION( 0x100000, "ymz", 0 ) // YMZ280B sound ROM on sub board
 	ROM_LOAD( "639jaa02.bin",  0x000000, 0x100000, CRC(c6163818) SHA1(b6f8f2d808b98610becc0a5be5443ece3908df0b) )
 
 	DISK_REGION( "cdrom" ) // Need a re-image
 	DISK_IMAGE_READONLY( "639jad01", 0, BAD_DUMP SHA1(39d41d5a9d1c40636d174c8bb8172b1121e313f8) )
 ROM_END
 
-#if 0 // FIXME
+/*
+// FIXME
```

Comment:
Once again, please don’t mess with stuff like this.

Reply:
Done, thx!